views/site/login.php

```diff
@@ -1,8 +1,10 @@
 <?php
 
-/* @var $this yii\web\View */
-/* @var $form yii\bootstrap\ActiveForm */
-/* @var $model app\models\LoginForm */
+/**
+ * @var \yii\web\View $this
+ * @var \yii\bootstrap\ActiveForm $form
+ * @var \app\models\LoginForm $model
+ */
```

Comment:
its a proposal, don't take it too serious. We'll keep the style that has better IDE support until there is a standard that we can refer to when asking IDEs to support something.